packages/docs/migrate-from-mxgraph.md

```diff
@@ -1,43 +1,344 @@
+```
+
+### `mxSvgCanvas2D`
+
+The `mxSvgCanvas2D` class has been renamed to `SvgCanvas2D` in `maxGraph`.
+
+#### Constructor
+
+The constructor parameter has been updated. Instead of accepting an `Element`, it now expects a `SvgElement` and a `boolean` value.
+
+**Before**
+```typescript
+// Old constructor
+const canvas = new mxgraph.mxSvgCanvas2D(element);
+```
+
+**Now**
+```typescript
+// Updated constructor
+const canvas = new SvgCanvas2D(svgElement, oneBoolean);
+```
+
+#### `getAlternateText()`
+
+change types ???????
+
+#### `format()`
+
+The `value` parameter, which was previously of type `string`, is now of type `number`.
+
+**Before**
+```typescript
+format:(value: string) => number
+```
+
+**Now**
+```typescript
+format:(value: number) => number
+```
+
+
+### `mxGraph`
+
+The `mxGraph` class has been renamed to `Graph` in `maxGraph`.
+There have also been some changes related to properties and methods.
+
+#### `panningHandler` property
 
-### TypeScript
+The `panningHandler` property has been removed and replaced by a plugin. Instead of accessing `panningHandler` directly, you can use the `getPlugin()` method to get the `PanningHandler` plugin instance. Here's an example:
 
-Remove the
-- `@typed-mxgraph/typed-mxgraph` dependency in the `package.json` file
-- `typeroots` settings related to `typed-mxgraph` in the `tsconfig.json` file
+**Before**
+```typescript
+// Old way to access panningHandler
+const panningHandler = graph.panningHandler;
+```
 
-As an example, you can check [this project](https://github.com/maxGraph/maxgraph-integration-examples/tree/main/projects/rollup-ts) which has been migrated
-from a [typed-mxgraph example](https://github.com/typed-mxgraph/typed-mxgraph-example-bundled-with-rollup).
+**Now**
+```typescript
+// Updated way using getPlugin()
+const panningHandler = this.graph.getPlugin('PanningHandler') as PanningHandler;
+```
 
+#### `getModel()`
 
-## General guidelines
+Instead of calling `getModel()`, you can directly access the `model` property of the graph object. Here's an example:
 
-- The names of _mxGraph_ objects were all prefixed with `mx`. The prefix has been dropped in _maxGraph_.
-- Most names remain the same.
-- Some utility functions, whose implementation is natively available in modern versions of ECMAScript, have been removed.
+**Before**
+```typescript
+// Old way of accessing the model
+const model = graph.getModel();
+```
 
+**Now**
+```typescript
+// Updated way to access the model
+const model = graph.model;
+```
 
-## Style related properties
+#### `insertVertex()` and `insertEdge()`
 
-Property rename
-- `autosize` into `autoSize` (as of maxgraph@0.2.0)
+The `insertVertex()` and `insertEdge()` methods in `maxGraph` now accept one object as parameter instead of multiple parameters. Instead of passing individual parameters, you can pass an object containing all the required properties. Here's an example:
```

Comment:
**nitpick:** The 2 methods access both the new form and the old form.
There is no need to migrate them